pom.xml

```diff
@@ -83,6 +83,11 @@
         <version>1.32</version>
       </dependency>
       <dependency>
+        <groupId>com.google.code.gson</groupId>
+        <artifactId>gson</artifactId>
+        <version>2.2.4</version>
```

Comment:
I think I did pull that version from Accumuo or Hadoop's pom.  I will try upping it.